### PR TITLE
Cookiecutter improvements: version, cookiecutter, folder error

### LINKED ISSUE #75: Provide better error message when outside directory
When outside the directory with the `.invenio` file. Most `invenio-cli` commands output:

```
ERROR:root:No flavour specified.
```

which is not as clear as `Missing .invenio file in current directory`. Use that error message instead.

### LINKED ISSUE #82: print invenio-cli version with invenio-cli --version
RIght now `invenio-cli --version` prints

```
Python 3.6.8
Flask 1.1.1
Werkzeug 0.16.0
```

It should print:

```
invenio-cli <version> 
```

### LINKED ISSUE #100: helpers [cookiecutter]: make cookiecutter version configurable
To avoid hardcoding the cookiecutter version/branch we could provide a feature to load from config var also.

My idea is to:

- For monthly releases we still pin and use the correct version.
- If a user wants to use another version/tag (e.g. us for development) it can be done.

### PR DESCRIPTION
**Pull request by fenekku**
closes #100 
closes #75 
closes #82

### CODE REVIEW

File: invenio_cli/cli.py [hunk 1]
```diff
@@ -20,19 +20,22 @@
...
 def cli():
     """Initialize CLI context."""
 
 
 @cli.command()
 @click.option('--flavour', type=click.Choice(['RDM'], case_sensitive=False),
               default='RDM', required=False)
-def init(flavour):
+@click.option('--cookiecutter', nargs=2, required=False,
+              help='Pass <cookiecutter git url> <branch> to select cookiecutter template')  # noqa
```
Comment (zzacharo) [inline invenio_cli/cli.py]: Is the url really needed? Is this intended to test cookiecutter fork urls?

File: invenio_cli/helpers/cookiecutter_wrapper.py [hunk 2]
```diff
@@ -23,12 +23,34 @@
...
         self.template_name = None
         self.flavour = flavour
 
+        if self.flavour.upper() == 'RDM':
+            self.template = (
+                cookiecutter[0] or
+                'https://github.com/inveniosoftware/'
+                'cookiecutter-invenio-rdm.git'
+            )
+            self.template_name = self.extract_template_name(self.template)
```
Comment (zzacharo) [inline invenio_cli/helpers/cookiecutter_wrapper.py]: Where is the `template_name` used/needed?

File: invenio_cli/cli.py (hunk 1 above)
Comment (ppanero) [reply invenio_cli/cli.py]: For forks is a good idea. I see a potential problem on people testing random stuff, but this is meant for us and many times we have stuff on our fork...

A quick minor:

```suggestion
              help='Pass <cookiecutter git url> <branch/tag> to select cookiecutter template')  # noqa
```

File: invenio_cli/helpers/cookiecutter_wrapper.py (hunk 2 above)
Comment (ppanero) [reply invenio_cli/helpers/cookiecutter_wrapper.py]: It's used on the replay part. It does not appear here cuz there is no changes, if you open the full file is at the bottom :)

File: invenio_cli/cli.py (hunk 1 above)
Comment (fenekku) [reply invenio_cli/cli.py]: Ah I see! Making it work for local cookiecutters would be even more useful. I will do that.

File: invenio_cli/helpers/cookiecutter_wrapper.py (hunk 2 above)
Comment (fenekku) [reply invenio_cli/helpers/cookiecutter_wrapper.py]: What he said :)

**Pull request merged**